src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/AadPlugin.cs

```diff
@@ -293,9 +297,19 @@ public Task<WebTokenRequest> CreateWebTokenRequestAsync(WebAccountProvider provi
 
             request.Properties.Add("wam_compat", "2.0");
 
+            AddTelemetryPropertiesToRequest(request);
+
             return Task.FromResult(request);
         }
 
+        private void AddTelemetryPropertiesToRequest(WebTokenRequest request)
```

Comment:
Maybe add these params as part of `WamAdapters.AddMsalParamsToRequest` to avoid adding them in aad / msa plugins separately.

Reply:
Done.